packages/opencascade/src/jupytercad.opencascade.d.ts

```diff
@@ -0,0 +1 @@
+export default null as any;
```

Comment:
What is the purpose of this? Is this for TypeScript to copy over the js file into `lib/`?

Reply:
This file is there to make tsc happy and compile without buiding occ.js into 'src'. It is not copied to `lib`. The real file in `lib` is provided by webpack in the occ build process.